package.json

```diff
@@ -59,8 +59,8 @@
     "@types/request": "^2.47.0",
     "@types/through2": "^2.0.33",
     "assert-rejects": "^1.0.0",
-    "gaxios": "^1.2.2",
     "codecov": "^3.0.4",
+    "gaxios": "^1.6.0",
```

Comment:
@JustinBeckwith gaxios is a devDependency here, but we require it in source code. Wat?

Reply:
That's really bad.  I would suggest against that 😆

Reply:
But that's how it is on master, I didn't add it! So, my thing is more like, what the heck? I'll change it as a dependency.